one_fm/api/doc_methods/hd_ticket.py

```diff
@@ -5,6 +5,10 @@
 
 def validate_hd_ticket(doc,event):
     if not doc.is_new():
+        default_support_team = frappe.get_value("Bug Buster Employee",None,'default_support_team')
+        if default_support_team:
+            if not doc.agent_group:
+                doc.agent_group = default_support_team
```

Comment:
It can be write like
```
if not doc.agent_group:
  default_support_team = frappe.db.get_single_value("Bug Buster Employee", "default_support_team")
  if default_support_team:
    doc.agent_group = default_support_team
```

Here we dont want to fetch data from server if agent_group exist in the doc

Reply:
@pjamsheer Thank you. I have effected the change.